packages/dd-trace/test/appsec/iast/analyzers/sql-injection-analyzer.mysql.plugin.spec.js

```diff
@@ -8,52 +11,123 @@ const vulnerabilityReporter = require('../../../../src/appsec/iast/vulnerability
 
 describe('sql-injection-analyzer with mysql', () => {
   let mysql
-  let connection
   withVersions('mysql', 'mysql', version => {
     prepareTestServerForIast('mysql', (testThatRequestHasVulnerability, testThatRequestHasNoVulnerability) => {
+      const srcFilePath = path.join(__dirname, 'resources', 'sql-injection-methods.js')
```

Comment:
I'm not sure why we need to use temp folders and fs calls here. It's good practice to really avoid those as they add complexity when it's not always needed. Is there an alternative solution here ?

Reply:
We are doing this kind of things to check that we are detecting the vulnerability in the correct file/line.
As we are excluding all the files in `dd-trace` directory, the code in the repository is in excluded files, so we are moving some files to temporal directories to check that the file/line is correct.

Reply:
Aaah, make sense now my bad :+1: